extension/src/debug.ts

```diff
@@ -184,16 +168,12 @@ class LispLaunchConfigurationProvider implements vscode.DebugConfigurationProvid
                 return undefined;
             } else {
                 let params = "";
-                if (config["attributes"]) {
-                    params = config["attributes"]["params"] ? config["attributes"]["params"] : "";
-                }
-                else {
-                    let text = getExtensionSettingString(LAUNCH_PARM);
-                    if (text)
-                        params = text;
-                }
+                let text = getExtensionSettingString(LAUNCH_PARM);
```

Comment:
similarly, if getExtensionSettingString returns "", we can just
`ProcessPathCache.globalParameter = getExtensionSettingString(LAUNCH_PARM);`